packages/core/data-transfer/src/strapi/providers/remote-destination/index.ts

```diff
@@ -143,9 +181,20 @@ class RemoteStrapiDestinationProvider implements IDestinationProvider {
             return callback(streamError);
           }
         }
-        const e = await this.#endStep(step);
+        const { error, stats } = await this.#endStep(step);
+
+        if (
+          stats &&
+          (stats.started !== this.stats[step].count || stats.finished !== this.stats[step].count)
+        ) {
+          callback(
+            new Error(
+              `Data missing: sent ${this.stats[step].count} ${step}, recieved ${stats.started} and added ${stats.finished} ${step}`
```

Comment:
```suggestion
              `Data missing: sent ${this.stats[step].count} ${step}, received ${stats.started} and added ${stats.finished} ${step}`
```

Not sure about the meaning of `received` and `added` here? :thinking:

Reply:
recieved as in arrived to remote, added as in successfuly transfered /inserted to the DB

Reply:
Alright, then `received` makes sense but maybe "saved" could be easier to understand for the user?